### PR TITLE
new icon: kdeneon (original,plain)

### PR DESCRIPTION
**Pull request by dotarjun**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

#1754 
## Link to prove your SVG is correct and up-to-date.

https://en.wikipedia.org/wiki/KDE_neon

### CODE REVIEW

File: icons/kdeneon/kdeneon-original.svg
Comment (Gautam25Raj) [inline icons/kdeneon/kdeneon-original.svg:1]: Optimized with [SVGViewer](https://svgviewer.dev/).

```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128"><defs><linearGradient id="h" gradientUnits="userSpaceOnUse" x1="25" y1="1004.362" x2="25" y2="1052.362" gradientTransform="translate(0 -2678.3) scale(2.66667)"><stop offset="0" style="stop-color:#1abc9c;stop-opacity:1"/><stop offset="1" style="stop-color:#2980b9;stop-opacity:1"/></linearGradient><linearGradient id="c" gradientUnits="userSpaceOnUse" x1="24" y1="1028.362" x2="49" y2="1028.362" gradientTransform="rotate(45 4083.407 1707.592) scale(3.29412)"><stop offset="0" style="stop-color:#000;stop-opacity:1"/><stop offset="1" style="stop-color:#000;stop-opacity:0"/></linearGradient><linearGradient id="e" gradientUnits="userSpaceOnUse" x1="24" y1="1028.362" x2="49" y2="1028.362" gradientTransform="rotate(45 11494.91 4777.53) scale(3.29412 9.26466)"><stop offset="0" style="stop-color:#000;stop-opacity:1"/><stop offset="1" style="stop-color:#000;stop-opacity:0"/></linearGradient><linearGradient id="g" gradientUnits="userSpaceOnUse" x1="24" y1="1028.362" x2="49" y2="1028.362" gradientTransform="rotate(45 17372.966 7212.296) scale(3.29412 13.99989)"><stop offset="0" style="stop-color:#000;stop-opacity:1"/><stop offset="1" style="stop-color:#000;stop-opacity:0"/></linearGradient><clipPath id="b"><path d="M0 0h128v128H0z"/></clipPath><clipPath id="d"><path d="M0 0h128v128H0z"/></clipPath><clipPath id="f"><path d="M0 0h128v128H0z"/></clipPath><mask id="j"><g filter="url(#a)"><path style="fill:#000;fill-opacity:.2;stroke:none" d="M0 0h128v128H0z"/></g></mask><mask id="l"><g filter="url(#a)"><path style="fill:#000;fill-opacity:.2;stroke:none" d="M0 0h128v128H0z"/></g></mask><mask id="n"><g filter="url(#a)"><path style="fill:#000;fill-opacity:.2;stroke:none" d="M0 0h128v128H0z"/></g></mask><g id="i" clip-path="url(#b)"><path style="stroke:none;fill-rule:evenodd;fill:url(#c)" d="M73.316 54.684 54.684 73.316l44.27 44.266a63.975 63.975 0 0 0 18.609-18.656Zm0 0"/></g><g id="k" clip-path="url(#d)"><path style="stroke:none;fill-rule:evenodd;fill:url(#e)" d="m90.203 37.797-1.176 1.176A35.422 35.422 0 0 1 99.41 64c0 19.559-15.851 35.41-35.41 35.41a35.422 35.422 0 0 1-25.027-10.383l-1.176 1.176 36.887 36.89c26.832-4.573 47.84-25.62 52.359-52.464Zm0 0"/></g><g id="m" clip-path="url(#f)"><path style="stroke:none;fill-rule:evenodd;fill:url(#g)" d="M103.598 24.39v.012l-3.078 3.078A51.647 51.647 0 0 1 115.676 64c0 28.54-23.137 51.676-51.676 51.676a51.667 51.667 0 0 1-36.52-15.16l-3.078 3.082 21.832 21.836A64.06 64.06 0 0 0 64 128c35.348 0 64-28.652 64-64a64.071 64.071 0 0 0-2.543-17.75Zm0 0"/></g><filter id="a" filterUnits="objectBoundingBox" x="0%" y="0%" width="100%" height="100%"><feColorMatrix in="SourceGraphic" values="0 0 0 0 1 0 0 0 0 1 0 0 0 0 1 0 0 0 1 0"/></filter></defs><path style="stroke:none;fill-rule:evenodd;fill:url(#h)" d="M128 64c0 35.348-28.652 64-64 64S0 99.348 0 64 28.652 0 64 0s64 28.652 64 64Zm0 0"/><use xlink:href="#i" mask="url(#j)"/><use xlink:href="#k" mask="url(#l)"/><use xlink:href="#m" mask="url(#n)"/><path style="stroke:none;fill-rule:evenodd;fill:#fcfcfc;fill-opacity:1" d="M64 8a4.61 4.61 0 0 0-4.496 3.605 52.7 52.7 0 0 0-29.387 12.2 4.616 4.616 0 0 0-2.351-.653 4.615 4.615 0 0 0-4.614 4.614c.004.828.227 1.636.649 2.347a52.72 52.72 0 0 0-12.219 29.399A4.612 4.612 0 0 0 8 64a4.61 4.61 0 0 0 3.605 4.496 52.7 52.7 0 0 0 12.2 29.387 4.616 4.616 0 0 0-.653 2.351 4.615 4.615 0 0 0 4.614 4.614 4.643 4.643 0 0 0 2.34-.645 52.743 52.743 0 0 0 29.406 12.219A4.613 4.613 0 0 0 64 120c2.16 0 4.027-1.5 4.496-3.61a52.723 52.723 0 0 0 29.426-12.171c.7.41 1.5.625 2.312.629a4.615 4.615 0 0 0 4.614-4.614 4.647 4.647 0 0 0-.649-2.347 52.712 52.712 0 0 0 12.266-29.407A4.62 4.62 0 0 0 120 64c0-2.16-1.5-4.027-3.61-4.496a52.705 52.705 0 0 0-12.175-29.422 4.608 4.608 0 0 0 .633-2.316 4.615 4.615 0 0 0-4.614-4.614 4.647 4.647 0 0 0-2.347.649A52.712 52.712 0 0 0 68.48 11.535 4.613 4.613 0 0 0 64 8Zm0 1.977a2.636 2.636 0 1 1-2.637 2.636A2.636 2.636 0 0 1 64 9.977Zm-4.336 4.187a4.604 4.604 0 0 0 8.648.063 50.065 50.065 0 0 1 27.786 11.511 4.608 4.608 0 0 0 6.172 6.16 50.057 50.057 0 0 1 11.523 27.778 4.606 4.606 0 0 0-.023 8.637 50.033 50.033 0 0 1-11.508 27.785 4.608 4.608 0 0 0-6.164 6.168 50.078 50.078 0 0 1-27.774 11.527A4.61 4.61 0 0 0 64 110.777a4.608 4.608 0 0 0-4.313 2.993 50.1 50.1 0 0 1-27.804-11.465 4.608 4.608 0 0 0-6.148-6.207 50.078 50.078 0 0 1-11.57-27.762 4.604 4.604 0 0 0 .066-8.648 50.077 50.077 0 0 1 11.468-27.805 4.608 4.608 0 0 0 6.203-6.152 50.045 50.045 0 0 1 27.762-11.567ZM64 23.813a4.606 4.606 0 0 0-4.504 3.64C41.027 29.723 27.141 45.395 27.106 64c.027 18.613 13.917 34.293 32.394 36.563a4.606 4.606 0 0 0 9.004-.016c18.469-2.27 32.355-17.942 32.39-36.547-.027-18.613-13.917-34.293-32.394-36.563a4.609 4.609 0 0 0-4.5-3.625Zm-36.234 1.316a2.635 2.635 0 1 1-.004 5.27 2.635 2.635 0 0 1 .004-5.27Zm72.468 0a2.636 2.636 0 1 1 .002 5.271 2.636 2.636 0 0 1-.002-5.271ZM64 25.789a2.635 2.635 0 1 1 0 5.27 2.636 2.636 0 1 1 0-5.27Zm-4.309 4.277A4.616 4.616 0 0 0 64 33.036a4.617 4.617 0 0 0 4.3-2.962C85.399 32.238 98.228 46.766 98.259 64c-.028 17.234-12.852 31.766-29.95 33.934A4.616 4.616 0 0 0 64 94.964a4.617 4.617 0 0 0-4.3 2.962C42.601 95.762 29.772 81.234 29.741 64c.028-17.234 12.852-31.766 29.95-33.934ZM64 50.824c-7.277 0-13.176 5.899-13.176 13.176S56.723 77.176 64 77.176 77.176 71.277 77.176 64 71.277 50.824 64 50.824Zm-51.387 10.54A2.635 2.635 0 0 1 15.246 64a2.635 2.635 0 1 1-5.27 0 2.636 2.636 0 0 1 2.637-2.637Zm102.774 0A2.636 2.636 0 0 1 118.023 64a2.636 2.636 0 1 1-2.636-2.637ZM64 96.94a2.636 2.636 0 1 1-2.637 2.637A2.636 2.636 0 0 1 64 96.941Zm-36.234.66a2.634 2.634 0 1 1-.003 5.269 2.634 2.634 0 0 1 .003-5.268Zm72.468 0a2.635 2.635 0 1 1 .004 5.27 2.635 2.635 0 0 1-.004-5.27ZM64 112.755a2.635 2.635 0 1 1 0 5.27 2.636 2.636 0 1 1 0-5.27Zm0 0"/></svg>
```

File: icons/kdeneon/kdeneon-plain.svg
Comment (Gautam25Raj) [inline icons/kdeneon/kdeneon-plain.svg:1]: Optimized with [SVGViewer](https://svgviewer.dev/).

```
<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><defs><clipPath id="b"><path d="M0 0h128v128H0z"/></clipPath><clipPath id="d"><path d="M0 0h128v128H0z"/></clipPath><clipPath id="f"><path d="M0 0h128v128H0z"/></clipPath><mask id="c"><g filter="url(#a)"><path style="fill:#000;fill-opacity:.2;stroke:none" d="M0 0h128v128H0z"/></g></mask><mask id="e"><g filter="url(#a)"><path style="fill:#000;fill-opacity:.2;stroke:none" d="M0 0h128v128H0z"/></g></mask><mask id="g"><g filter="url(#a)"><path style="fill:#000;fill-opacity:.2;stroke:none" d="M0 0h128v128H0z"/></g></mask><filter id="a" filterUnits="objectBoundingBox" x="0%" y="0%" width="100%" height="100%"><feColorMatrix in="SourceGraphic" values="0 0 0 0 1 0 0 0 0 1 0 0 0 0 1 0 0 0 1 0"/></filter></defs><path style="stroke:none;fill-rule:evenodd;fill:#fff" d="M128 64c0 35.348-28.652 64-64 64S0 99.348 0 64 28.652 0 64 0s64 28.652 64 64Zm0 0"/><g clip-path="url(#b)" mask="url(#c)"><path style="stroke:none;fill-rule:evenodd;fill:#fff" d="M73.316 54.684 54.684 73.316l44.27 44.266a63.975 63.975 0 0 0 18.609-18.656Zm0 0"/></g><g clip-path="url(#d)" mask="url(#e)"><path style="stroke:none;fill-rule:evenodd;fill:#fff" d="m90.203 37.797-1.176 1.176A35.422 35.422 0 0 1 99.41 64c0 19.559-15.851 35.41-35.41 35.41a35.422 35.422 0 0 1-25.027-10.383l-1.176 1.176 36.887 36.89c26.832-4.573 47.84-25.62 52.359-52.464Zm0 0"/></g><g clip-path="url(#f)" mask="url(#g)"><path style="stroke:none;fill-rule:evenodd;fill:#fff" d="M103.598 24.39v.012l-3.078 3.078A51.647 51.647 0 0 1 115.676 64c0 28.54-23.137 51.676-51.676 51.676a51.667 51.667 0 0 1-36.52-15.16l-3.078 3.082 21.832 21.836A64.06 64.06 0 0 0 64 128c35.348 0 64-28.652 64-64a64.071 64.071 0 0 0-2.543-17.75Zm0 0"/></g><path style="fill-rule:evenodd;fill-opacity:1;fill:#fff;stroke:#000" d="M64 8a4.61 4.61 0 0 0-4.496 3.605 52.7 52.7 0 0 0-29.387 12.2 4.616 4.616 0 0 0-2.351-.653 4.615 4.615 0 0 0-4.614 4.614c.004.828.227 1.636.649 2.347a52.72 52.72 0 0 0-12.219 29.399A4.612 4.612 0 0 0 8 64a4.61 4.61 0 0 0 3.605 4.496 52.7 52.7 0 0 0 12.2 29.387 4.616 4.616 0 0 0-.653 2.351 4.615 4.615 0 0 0 4.614 4.614 4.643 4.643 0 0 0 2.34-.645 52.743 52.743 0 0 0 29.406 12.219A4.613 4.613 0 0 0 64 120c2.16 0 4.027-1.5 4.496-3.61a52.723 52.723 0 0 0 29.426-12.171c.7.41 1.5.625 2.312.629a4.615 4.615 0 0 0 4.614-4.614 4.647 4.647 0 0 0-.649-2.347 52.712 52.712 0 0 0 12.266-29.407A4.62 4.62 0 0 0 120 64c0-2.16-1.5-4.027-3.61-4.496a52.705 52.705 0 0 0-12.175-29.422 4.608 4.608 0 0 0 .633-2.316 4.615 4.615 0 0 0-4.614-4.614 4.647 4.647 0 0 0-2.347.649A52.712 52.712 0 0 0 68.48 11.535 4.613 4.613 0 0 0 64 8Zm0 1.977a2.636 2.636 0 1 1-2.637 2.636A2.636 2.636 0 0 1 64 9.977Zm-4.336 4.187a4.604 4.604 0 0 0 8.648.063 50.065 50.065 0 0 1 27.786 11.511 4.608 4.608 0 0 0 6.172 6.16 50.057 50.057 0 0 1 11.523 27.778 4.606 4.606 0 0 0-.023 8.637 50.033 50.033 0 0 1-11.508 27.785 4.608 4.608 0 0 0-6.164 6.168 50.078 50.078 0 0 1-27.774 11.527A4.61 4.61 0 0 0 64 110.777a4.608 4.608 0 0 0-4.313 2.993 50.1 50.1 0 0 1-27.804-11.465 4.608 4.608 0 0 0-6.148-6.207 50.078 50.078 0 0 1-11.57-27.762 4.604 4.604 0 0 0 .066-8.648 50.077 50.077 0 0 1 11.468-27.805 4.608 4.608 0 0 0 6.203-6.152 50.045 50.045 0 0 1 27.762-11.567ZM64 23.813a4.606 4.606 0 0 0-4.504 3.64C41.027 29.723 27.141 45.395 27.106 64c.027 18.613 13.917 34.293 32.394 36.563a4.606 4.606 0 0 0 9.004-.016c18.469-2.27 32.355-17.942 32.39-36.547-.027-18.613-13.917-34.293-32.394-36.563a4.609 4.609 0 0 0-4.5-3.625Zm-36.234 1.316a2.635 2.635 0 1 1-.004 5.27 2.635 2.635 0 0 1 .004-5.27Zm72.468 0a2.636 2.636 0 1 1 .002 5.271 2.636 2.636 0 0 1-.002-5.271ZM64 25.789a2.635 2.635 0 1 1 0 5.27 2.636 2.636 0 1 1 0-5.27Zm-4.309 4.277A4.616 4.616 0 0 0 64 33.036a4.617 4.617 0 0 0 4.3-2.962C85.399 32.238 98.228 46.766 98.259 64c-.028 17.234-12.852 31.766-29.95 33.934A4.616 4.616 0 0 0 64 94.964a4.617 4.617 0 0 0-4.3 2.962C42.601 95.762 29.772 81.234 29.741 64c.028-17.234 12.852-31.766 29.95-33.934ZM64 50.824c-7.277 0-13.176 5.899-13.176 13.176S56.723 77.176 64 77.176 77.176 71.277 77.176 64 71.277 50.824 64 50.824Zm-51.387 10.54A2.635 2.635 0 0 1 15.246 64a2.635 2.635 0 1 1-5.27 0 2.636 2.636 0 0 1 2.637-2.637Zm102.774 0A2.636 2.636 0 0 1 118.023 64a2.636 2.636 0 1 1-2.636-2.637ZM64 96.94a2.636 2.636 0 1 1-2.637 2.637A2.636 2.636 0 0 1 64 96.941Zm-36.234.66a2.634 2.634 0 1 1-.003 5.269 2.634 2.634 0 0 1 .003-5.268Zm72.468 0a2.635 2.635 0 1 1 .004 5.27 2.635 2.635 0 0 1-.004-5.27ZM64 112.755a2.635 2.635 0 1 1 0 5.27 2.636 2.636 0 1 1 0-5.27Zm0 0"/></svg>
```

File: devicon.json
Comment (Gautam25Raj) [inline devicon.json:1]: Add `"altnames": []` after `"name"`.

**Review by Gautam25Raj — CHANGES_REQUESTED** [review]

File: icons/kdeneon/kdeneon-plain.svg
Comment (lunatic-fox) [inline icons/kdeneon/kdeneon-plain.svg:1]: A suggestion of `plain` version:

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
Maybe we can use this `plain` version that will be colored by `devicon.json` in `#21769a`.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🚀

**Review by lunatic-fox — APPROVED** [review]
Icons look great! ✔
Thank you for this contribution! 🚀
2023-06-02_21h37_57

**Review by Gautam25Raj — APPROVED** [review]
Great Work. 
Thank you for Contributing. 🚀

**Review by lunatic-fox — APPROVED** [review]

**Pull request merged**